.flake8

```diff
@@ -0,0 +1,24 @@
+[flake8]
+max-line-length=127
+count=true
+statistics=true
+
+ignore =
+    W605
+
+select =
+    E4
+    E7,
+    E9,
+    W6,
+    F821,
+    F822
```

Comment:
I need to go over these in another PR. I just grabbed them from what I normally used but haven't reviewed them in a while. See #6